src/pages/SocialGraph.vue

```diff
@@ -210,27 +217,51 @@ export default {
     },
 
     async fetchPageData(page = INITIAL_PAGE) {
+      if (!this.address) return [];
       const i = page - 1;
       const params = {
         'pagination.limit': INDEXER_QUERY_LIMIT,
         'pagination.offset': i * INDEXER_QUERY_LIMIT,
         reverse: true,
       };
-      params[this.paramField] = this.account;
+      params[this.paramField] = this.address;
       const { data } = await indexerApi.get(`/likechain/likenft/v1/${this.type}`, { params });
       const accountData = data[this.responseField] || [];
       const pageData = await this.aggregate(accountData);
       return pageData;
     },
 
     async load() {
+      if (isValidAddress(this.inputString)) {
+        this.address = this.inputString;
+      } else if (this.addressOfLikerId) {
+        this.address = this.addressOfLikerId;
```

Comment:
Strange to use computed for this, just call the getter in this function will do?